.eslintrc.cjs

```diff
@@ -77,6 +101,12 @@ module.exports = {
         '@typescript-eslint/consistent-type-imports': ['error'],
         // We choose to disable it and choose later if we want to enable it. See https://github.com/process-analytics/bpmn-visualization-js/pull/2821.
         '@typescript-eslint/consistent-type-definitions': 'off',
+        'import/no-unresolved': [
+          'error',
+          {
+            ignore: ['^bpmn-visualization$'],
```

Comment:
**question:** can you give me some information about the need for this (I didn't test without the configuration). May be interesting to add a comment for a later review and not forget the reason.

Reply:
I change the waybto fix the error. See 26caaee